plugin/storage/es/factory_test.go

```diff
@@ -245,7 +245,7 @@ func TestArchiveDisabled(t *testing.T) {
 func TestArchiveEnabled(t *testing.T) {
 	f := NewFactory()
 	f.primaryConfig = &escfg.Configuration{}
-	f.archiveConfig = &escfg.Configuration{Enabled: true}
+	f.archiveConfig = &escfg.Configuration{Enabled: true, UseReadWriteAliases: true}
```

Comment:
is this change necessary? It goes contrary to default config.

Reply:
@yurishkuro This was mostly for code coverage purposes. I can remove it but then we may need to merge with less than 100% code coverage

Reply:
I think my comment on the archive tests would address cove coverage

Reply:
@yurishkuro It didn't because that test doesn't go through `CreateArchiveSpanReader`. I just added a test for both cases here.